packages/aragon-wrapper/src/interfaces.js

```diff
@@ -20,15 +27,66 @@ const ABIS = {
   'aragon/Forwarder': abiAragonForwarder.abi,
   'aragon/Kernel': abiAragonKernel.abi,
   'aragon/EVM Script Registry': abiAragonEVMScriptRegistry.abi,
-  'aragon/Repo': abiAragonRepo.abi,
+  'apm/APM Registry': abiApmRegistry.abi,
+  'apm/Repo': abiApmRepo.abi,
+  'apm/ENS Subdomain Registrar': abiApmEnsSubdomainRegistrar.abi,
```

Comment:
What is the canonical source for the "ABI name", e.g. `ENS Subdomain Registrar`

Reply:
In other words, where do the strings come from in 
https://github.com/aragon/aragon.js/blob/1ea7047a064ca3796419c89a54c1a5224dee7af6/packages/aragon-wrapper/src/interfaces.js#L52-L60

Reply:
Those are mostly made up names we chose ourselves as the "human readable app name".

They would correlate to the name in an app's `manifest.json` (e.g. [Token Manager's](https://github.com/aragon/aragon-apps/blob/master/apps/token-manager/manifest.json#L2)).